src/main.rs

```diff
@@ -1,11 +1,12 @@
-mod init;
+mod commands;
+mod config;
 mod ss58;
 mod utils;
 
 use clap::Command;
-use init::init;
+use commands::{farm::farm, init::init};
 
-fn cli() -> Command<'static> {
```

Comment:
Wow. Could you rewrite it using this `clap` tutorial:
https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html

Reply:
rebase was not that successful it seems. This was removed... Will fix :)

Reply:
Nope. I meant, in general, using raw `clap` would lead to more code than using `clap::Parser`. So it would be better to switch to that. Monorepo and substrate use that everywhere.

Reply:
Oh got it. I did not use it on purpose. And I followed their cookbook on git example (https://docs.rs/clap/latest/clap/_derive/_cookbook/git/index.html).
The reason is, we don't want this: `subspace -farm`
but we want this: `subspace farm`
So I'm using commands instead of arguments, that was the reason.

Reply:
I believe you can achieve that using subcommands. Take a look at this:
https://github.com/clap-rs/clap/blob/master/examples/git-derive.rs

Reply:
wonderful, I'll close this PR and do a 2nd take on it then, thanks @i1i1 !